docs/contributing.md

```diff
@@ -28,16 +28,15 @@ Since Flarum is so extension-driven, we highly recommend [our extension docs](ex
 
 ### Setting Up a Local Codebase
 
-[flarum/flarum](https://github.com/flarum/flarum) is a "skeleton" application which uses Composer to download [flarum/core](https://github.com/flarum/core) and a [bunch of extensions](https://github.com/flarum). In order to work on these, we recommend forking and cloning them into a [Composer path repository](https://getcomposer.org/doc/05-repositories.md#path):
+[flarum/flarum](https://github.com/flarum/flarum) is a "skeleton" application which uses Composer to download the core package and a bunch of extensions. Source code for Flarum core, extensions, and all packages used by the aforementioned is located in the Flarum monorepo [flarum/framework](https://github.com/flarum/framework). In order to contribute to these, you'll need to fork and clone the monorepo repository locally, and then add it to your dev environment as a [Composer path repository](https://getcomposer.org/doc/05-repositories.md#path):
 
 ```bash
 git clone https://github.com/flarum/flarum.git
 cd flarum
 
-# Set up a Composer path repository for Flarum packages
-composer config repositories.0 path "packages/*"
-git clone https://github.com/<username>/core.git packages/core
-git clone https://github.com/<username>/tags.git packages/tags # etc
+# Set up a Composer path repository for Flarum monorepo packages
+composer config repositories.0 path "PATH_TO_MONOREPO/*/*"
+git clone https://github.com/<username>/framework.git packages/framework
```

Comment:
I think it's easier to have it as `packages` directly

```suggestion
composer config repositories.0 path "packages/*/*"
git clone https://github.com/<username>/framework.git packages/framework
```

Reply:
Not sure about that, actually. `packages` could refer to the new monorepo, or to a generic packages directory where all flarum extensions are installed.

Reply:
I was more referring to the fact that the command right after it clones the monorepo fork to the same packages path, so it's a no brainer copy past for new contributors.

Though I don't feel strongly about this, just a thought I had when I noticed the second command.

Will make them consistent.

Reply:
Ah but then it doesn't work, as `packages/*/*` would refer to the folders in the root level of `framework`, not the extensions/packages inside.

If we want to be very general, perhaps we should clone to `monorepos/flarum`, and then have the path be `monorepos/*/*/*`?

Reply:
Oh wait, I missed what you actually meant. True, it wouldn't be correctly pointing to the actual monorepo packages.

perhaps a simple change to `packages/framework/*/*`, even though this wouldn't take into consideration local 3rd party extensions, since it's not the point of the contributing guide I think it should be fine, for extension development we have this https://docs.flarum.org/extend/start#extension-packaging.

For the more general solution, I don't think there'll be many monorepos in the ecosystem, and even if there will be, they probably won't have the same subdirectory structure Flarum has, so not sure it's worth it.

Reply:
Let's just leave it as `PATH_TO_MONOREPO` 👍🏼, both commands are in sync now.